samples/stock/demo/relative-rotation-graph/demo.js

```diff
@@ -0,0 +1,370 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'scatter',
+        height: '90%',
+        plotBorderWidth: 10,
+        plotBorderColor: '#ffffff',
+        backgroundColor: '#f7f7f8',
+        spacing: [10, 85, 15, 20],
+        events: {
+            load: function () {
+                this.plotBorder.attr({
+                    rx: 10,
+                    ry: 10,
+                    zIndex: 6
+                });
+            }
```

Comment:
Since the rest of the chart is 100% config (and serializable to JSON), this function hack is kind of an eyesore. I think we should make it more like a plugin. Use `options.chart.className` to enable it, and add a global event listener. That way we keep the config clean and usable for all kinds of implementations.

`Highcharts.addEvent(Highcharts.Chart, 'load', function () {
    if (this.options.chart.className.indexOf('rounded-plot-border') !== -1) {
        ...
    }
});`